### ISSUE TITLE
Display arrays properly in the results

### ISSUE THREAD

**Issue by mobz** [mobz/elasticsearch-head#53]
It is better to see something representing the array than nothing at all.

In the Browser or Search Query tabs, complex types from the JSON such as arrays are not rendered at all in the result table.

When the code detects arrays, the easiest way is to concatenate all the values as a coma-separated list and render this as a string.
More elegant is to create a cell per element and span all the element cells them over a bigger cell representing the whole array.

**Comment by marcosscriven** [comment]
Just searching for open issues with the keyword 'browser' to suggest incorporating something like http://json2table.com/ 

I just copied and pasted a whole search result into it, and it seemed to do something quite sane.

**Comment by yzhang226** [comment]
yes, this issue occurs when any field is array , if use kibana, this array field can be seen.
can we think one way to solve this issue.

**Comment by Luxiyalu** [comment]
Still an issue. Doesn't look like repo is maintained anymore though.
